packages/contentstack-import/src/import/modules-js/workflows.js

```diff
@@ -147,6 +147,15 @@ module.exports = class importWorkflows {
         })
         .catch(function (error) {
           fileHelper.writeFileSync(workflowFailsPath, self.fails);
+          let message_content_type = '';
+          if (error.request !== undefined && JSON.parse(error.request.data).workflow !== undefined) {
+            if (JSON.parse(error.request.data).workflow.name) {
+              message_content_type =
+                ' Due to Workflow name - ' + JSON.parse(error.request.data).workflow.name;
```

Comment:
@cs-raj Can you please validate the error messages with the docs team?

Reply:
Validated with Varsha and made the changes